src/utils.ts

```diff
@@ -80,16 +80,28 @@ async function executeCommandBe(
     }
 }
 
-const compareNumbers = (actual: number, gte: number, lte: number, eq?: number): boolean => {
-    if (typeof eq === 'number') {
-        return actual === eq
+const compareNumbers = (actual: number, options: ExpectWebdriverIO.NumberOptions = {}): boolean => {
+    // Equals case
+    if (typeof options.eq == 'number') {
+        return actual == options.eq
```

Comment:
Let's try to parse the `actual` value to an integer and throw if this fails.

```suggestion
        return actual === options.eq
```

Reply:
Okay, so should I just add in the === here or do you want me to refactor it a bit more?